app/views/layouts/application.html.erb

```diff
@@ -32,16 +32,22 @@
         <div class="collapse navbar-collapse" id="navbarSupportedContent">
           <ul class="navbar-nav me-auto">
             <li class="nav-item active">
-              <%= link_to "Home", root_path, class: "nav-link" %>
+              <%= link_to "Home", root_path, class: "nav-link", style:" line-height: 40px;" %>
             </li>
             <li class="nav-item">
-              <%= link_to "Messages", players_path, class: "nav-link" %>
+              <%# <%= link_to "Messages", players_path, class: "nav-link" %>
+            <% if user_signed_in? %>
+              <%= link_to "Dashboard", dashboards_path, class: "nav-link", style:" line-height: 40px;" %>
+            <% else %>
+              <% "" %>
+            <% end %>
+
             </li>
-            <li class="nav-item dropdown">
-              <% if user_signed_in? %>
+            <li class="nav-item dropdown" style="margin-right50px">
+              <% if user_signed_in? && current_user.avatar.present? %>
                 <%= cl_image_tag(current_user.avatar.key, class: "avatar", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false) %>
               <% else %>
-                <%= link_to "Sign in", new_user_session_path, class: "nav-link" %>
+                <%= cl_image_tag("avatar_nybl5s", class: "avatar", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false) %>
```

Comment:
Ligne 40 à 50 : on vient de tester sur nos branches et l'image ne s'affiche pas, le sign in non plus du coup